app.py

```diff
@@ -64,39 +64,54 @@ def validate_arrangement(arrangement):
         arrangement_id = arrangement['_id']
         name = arrangement['name']
         timestamp = arrangement['timestamp']
+        modified_timestamp =arrangement['modified_timestamp']
         is_deleted = arrangement['is_deleted']
-        items = arrangement['items']
 
+        items = arrangement['items']
+        item_id_list = []
         for item in items:
             item_id = item['_id']
             item_name = item['name']
             item_size = item['size']
+            item_id_list.append(item_id)
             if item_id == "" or item_name == "" or item_size == "":
-                break
+                return False
 
         containers = arrangement['containers']
+        container_id_list = []
         for container in containers:
             container_id = container['_id']
             container_name = container['name']
             container_size = container['size']
+
+            container_id_list.append(container_id)
             if container_id == "" or container_name == "" or container_size == "":
-                break
+                return False
 
         snapshots = arrangement['snapshots']
+
         for snapshot in snapshots:
             snapshot_id = snapshot['_id']
             snapshot_name = snapshot['name']
             snapshot_dict = snapshot['snapshot']
             if snapshot_id == "" or snapshot_name == "" or snapshot_dict == "":
-                break
-            for key, value in snapshot_dict.items():
-                container_key_id = key
-
-                for item_value_id in value:
-                    if container_key_id == "" or not item_value_id:
-                        break
-
-        if(arrangement_id and name and timestamp and item_id
+                return False
+            else:
+                for key, value in snapshot_dict.items():
+                    container_key_id = key
+                    if container_key_id in container_id_list:
+                        if True:
```

Comment:
what does this do??

Reply:
this check loops through the key and value inside the snapshot dictionary. and set the logic as True if there the container_key_id is included inside the container id list.

Reply:
he's asking why you have `if True:`. That literally does nothing because it will always enter the `if` statement. Did you mean to check if a variable is true?

Reply:
@natyconnor  correct this in the last pull request #8 #